src/ray/raylet/format/node_manager.fbs

```diff
@@ -263,3 +265,8 @@ table SetResourceRequest {
   // Client ID where this resource will be set.
   client_id: string;
 }
+
+table SubscribePlasma {
```

Comment:
should we rename this (and related places) to SubscribePlasmaReady since we only subscribe for addition, not deletion?

Reply:
That makes sense, will do!